### PR TITLE
Create unlox-beta.rb

### PR DESCRIPTION
**Pull request by JounQin**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, if **adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not [already refused].
- [x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/Homebrew/homebrew-cask-versions/pulls
[already refused]: https://github.com/Homebrew/homebrew-cask-versions/search?q=is%3Aclosed&type=Issues
[the correct repo]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256

### CODE REVIEW

File: Casks/unlox-beta.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+cask 'unlox-beta' do
+  version '3.0.0.2'
+  sha256 'e0fb5f35b12d1bcfd553a63fdf5b2a4e85d910311a5f0ffa13538f41d7ba1da0'
+
+  url 'https://unlox.it/beta/download/'
+  name 'Unlox Beta'
+  homepage 'https://unlox.it/get'
```
Comment (vitorgalvao) [inline Casks/unlox-beta.rb]: Is there no page for the beta?

Comment (JounQin) [reply Casks/unlox-beta.rb]: I'm afraid not.

Comment (vitorgalvao) [reply Casks/unlox-beta.rb]: I don’t see the beta referenced on the page.

Comment (JounQin) [reply Casks/unlox-beta.rb]: 156FECBC-A56A-4C68-9079-74306ABA48F8

I get it from Apple's TestFlight.

Comment (vitorgalvao) [reply Casks/unlox-beta.rb]: Which means they‘re not making it publicly accessible to anyone on the website. I’d rather not add it, then. It’s not like the non-beta cask is popular.

**Pull request closed without merging**